NEWS.md

```diff
@@ -0,0 +1,55 @@
+
+# MMAction2 Release Notes
+
+## v0.6.0 (Master)
```

Comment:
Maybe just `Master`. We will tag 0.6.0 when it is ready. Before that there may be patch versions like 0.5.1.